src/Header/Header.tsx

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Header.css';
-import logo from '../assets/logo.png';
-import { Navbar, Nav, Button } from 'react-bootstrap';
-import { useHistory } from 'react-router-dom';
+import folderIcon from '../assets/headerIcons/folderIcon.svg';
```

Comment:
Try to import this with
`import { ReactComponent as FolderIcon } from '../assets/headerIcons/folderIcon.svg'`